util/src/main/resources/org/killbill/billing/util/migration/V20161105000000__session_id.sql

```diff
@@ -0,0 +1,2 @@
+alter table sessions add column id varchar(36) not null;
```

Comment:
Let's enforce the ordering (i.e. `after record_id`), as it makes it easier to debug exported data.